css/style.css

```diff
@@ -45,7 +45,7 @@
             font-size: 35px;
             color: white;
             text-align: center;
-            text-shadow: 5px 5px 10px black;
+            text-shadow: 5px 5px 10px rgb(14, 12, 12);
```

Comment:
Please make use of color codes instead of just plain colors as specified in the design